Gems/ROS2/Code/Include/ROS2/Lidar/LidarRegistrarBus.h

```diff
@@ -16,14 +16,15 @@ namespace ROS2
     //! Enum bitwise flags used to describe LidarSystem's feature support.
     enum LidarSystemFeatures : uint16_t
     {
-        None                    = 0,
-        Noise                   = 1,
-        CollisionLayers         = 1 << 1,
-        EntityExclusion         = 1 << 2,
-        MaxRangePoints          = 1 << 3,
-        PointcloudPublishing    = 1 << 4,
-        Intensity               = 1 << 5,
-        All                     = 0b1111111111111111,
+        None = 0,
+        Noise = 1,
+        CollisionLayers = 1 << 1,
+        EntityExclusion = 1 << 2,
+        MaxRangePoints = 1 << 3,
+        PointcloudPublishing = 1 << 4,
+        Intensity = 1 << 5,
+        Segmentation = 1 << 6,
+        All = 0b1111111111111111,
```

Comment:
Somehow the formatting (alignment) was lost, can you restore it?

Reply:
This is probably due to applying clang format

Reply:
I've added necessary changes. I've also set `All` to correspond to **_all features_** enabled in contrast to how it was previously - **_all bits_**.